include/ninshiki_cpp/config/grpc/config.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2024 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef NINSHIKI_CPP__CONFIG__GRPC__CONFIG_HPP_
+#define NINSHIKI_CPP__CONFIG__GRPC__CONFIG_HPP_
+
+#include <absl/flags/flag.h>
+#include <absl/flags/parse.h>
+#include <absl/strings/str_format.h>
+#include <grpc/support/log.h>
+#include <grpcpp/grpcpp.h>
+#include <ninshiki_cpp/config/grpc/call_data.hpp>
+#include <ninshiki_cpp/config/grpc/call_data_base.hpp>
+#include <ninshiki_cpp/detector/color_detector.hpp>
+#include <ninshiki_interfaces/msg/color_setting.hpp>
+#include <ninshiki_interfaces/ninshiki.grpc.pb.h>
+#include <ninshiki_interfaces/ninshiki.pb.h>
+#include <nlohmann/json.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <chrono>
+#include <fstream>
+#include <future>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+#include <thread>
```

Comment:
Some of these headers may be unnecessary, don't put all of them in the header file.